### PR TITLE
Mobile Punchlist

### PR DESCRIPTION
**Pull request by Georgehe4**
#558 
@devoncarew 
Not yet ready to merge.

Addressing some minor issues.
Page transitions still not working.

### CODE REVIEW

**Comment by devoncarew** [comment]
Re: transitions, I think they changed the way they're implemented from 0.5 to 1.0 - the old code referencing 0.5 likely won't do anything.

File: web/mobile.html [hunk 1]
```diff
@@ -22,7 +22,8 @@
     <link href='https://fonts.googleapis.com/css?family=Inconsolata:400,700'
           rel='stylesheet' type='text/css'>
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto">
-
+    <link rel="import" href="packages/dart_pad/bower/neon-animation/neon-animated-pages.html">
```
Comment (devoncarew) [inline web/mobile.html]: in use?

**Comment by devoncarew** [comment]
lgtm!

**Comment by Georgehe4** [comment]
Merging.

**Pull request merged**